modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/config/DefaultMutableConfigurationStats.java

```diff
@@ -80,7 +80,7 @@ public int getActiveComponents() {
   @Override
   public int addInflightOperation() {
     updateLastUsed();
```

Comment:
There is no need to call the "updateLastUsed" method on all the deprecated methods, since the add/discount activeComponente will end up updating it anyway.

Reply:
I thought the intention behind calling the add/discount method directly was to abstract from its implementation, but here we are using the knowledge that it also calls updateLastUsed. Also if I delete the line updateLastUsed it will be just like before de PR review only that we shifted some code around.

Reply:
I don't understand what you mean. This method needs to delegate to `addActiveComponent()` which will update the timestamp. I see no reason why `updateLastUsed()` should be called from here